neqo-transport/src/stream_id.rs

```diff
@@ -17,7 +17,7 @@ pub enum StreamType {
 }
 
 #[derive(Debug, Eq, PartialEq, Clone, Copy, Ord, PartialOrd, Hash)]
-pub struct StreamId(u64);
+pub struct StreamId(pub u64);
```

Comment:
Implement `From<u64>` as above, rather than making .0 pub.

Reply:
impl From<u64> is implemented for StreamId:
```
impl From<u64> for StreamId {
    fn from(val: u64) -> Self {
        Self::new(val)
    }
}
```
but without making that pub it won't compile (though note I haven't gotten the generic to work, so I'm still coercing into StreamId directly)

Reply:
Yeah, this isn't going to be good enough.  You can use the From implementation in the tests and not need this change.  But I think that making the test function better (along the lines I described) would be better yet.